app/components/pending-record/component.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
```

Comment:
currently, they are almost same, but in the future, we will have different wording, requirements for example accepted and rejected will require info from the reviews log and reviewer name